### PR TITLE
Docker setting fix for right click scenario

### PR DESCRIPTION
**Pull request by tiwarishub**
Fix for issue https://github.com/github/azure-integration/issues/63

### CODE REVIEW

File: src/configure/configure.ts [hunk 1]
```diff
@@ -556,17 +556,20 @@ class Orchestrator {
...
-            .filter((template) => template.templateType === TemplateType.REMOTE)
-            .map((template: RemotePipelineTemplate) => template.workingDirectory.toLowerCase())));
-        const applicationSettings = repoAnalysisResult.applicationSettingsList.filter((applicationSetting) => {
-            if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.REMOTE) {
-                return workingDirectories.indexOf(applicationSetting.settings.workingDirectory.toLowerCase()) >= 0;
-            }
-            return applicationSetting.language === this.inputs.pipelineConfiguration.template.language;
 
-        });
+        let applicationSettings: ApplicationSettings[] = repoAnalysisResult.applicationSettingsList;
```
Comment (anuragc617) [inline src/configure/configure.ts:560]: Is it working after this change?

**Review by anuragc617 — APPROVED** [review]

File: src/configure/configure.ts (hunk 1 above)
Comment (tiwarishub) [reply src/configure/configure.ts:560]: Yes, it worked.

File: src/configure/configure.ts [hunk 2]
```diff
@@ -556,17 +556,20 @@ class Orchestrator {
...
+        if (!this.isResourceAlreadySelected()) {
+            const workingDirectories = Array.from(new Set(this.inputs.potentialTemplates
+                .filter((template) => template.templateType === TemplateType.REMOTE)
```
Comment (bishal-pdMSFT) [inline src/configure/configure.ts:563]: @anuragc617 one question:  should we not be filtering more than just remote templates. We should only be interested in working directories of the templates which match the selected template attributes.

File: src/configure/configure.ts [hunk 3]
```diff
@@ -556,17 +556,20 @@ class Orchestrator {
...
+                .map((template: RemotePipelineTemplate) => template.workingDirectory.toLowerCase())));
+            applicationSettings = repoAnalysisResult.applicationSettingsList.filter((applicationSetting) => {
+                if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.REMOTE) {
```
Comment (bishal-pdMSFT) [inline src/configure/configure.ts:566]: Could we have altered the fix to rather check for both Remote template as well valid working directory?

**Pull request merged**